diff --git a/BUILD.gn b/BUILD.gn
index e4a82bc8d6..67831c8bd8 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -41,6 +41,7 @@ if (!build_with_chromium) {
     deps = [ ":webrtc" ]
     if (rtc_build_examples) {
       deps += [ "examples" ]
+      deps += [ "libwebrtc/examples" ]
     }
     if (rtc_build_tools) {
       deps += [ "rtc_tools" ]
@@ -521,11 +522,6 @@ if (!build_with_chromium) {
   # Target to build all the WebRTC production code.
   rtc_static_library("webrtc") {
     # Only the root target and the test should depend on this.
-    visibility = [
-      "//:default",
-      "//:webrtc_lib_link_test",
-    ]
-
     sources = []
     complete_static_lib = true
     suppressed_configs += [ "//build/config/compiler:thin_archive" ]
diff --git a/examples/BUILD.gn b/examples/BUILD.gn
index 69022724e4..b1f807b9e2 100644
--- a/examples/BUILD.gn
+++ b/examples/BUILD.gn
@@ -49,8 +49,8 @@ group("examples") {
   if (is_linux || is_chromeos || is_win) {
     deps += [
       ":peerconnection_server",
-      ":stunserver",
-      ":turnserver",
+#      ":stunserver",
+#      ":turnserver",
     ]
     if (current_os != "winuwp") {
       deps += [ ":peerconnection_client" ]
